### ISSUE TITLE
Remove warning for using categorical units to plot a list of strings for histogram generation

### ISSUE THREAD

**Issue by bobleesj** [bobleesj/cifkit#11]
**Checklist**
- [x] Are you using the latest stable version?

**Describe the bug**

For the histogram generations with coordination numbers, the following warning should be removed.
```
2024-07-06 11:31:18,905 - INFO - Using categorical units to plot a list of strings that are all parsable as floats or dates. If these strings should be plotted as numbers, cast to the appropriate data type before plotting.
2024-07-06 11:31:18,905 - INFO - Using categorical units to plot a list of strings that are all parsable as floats or dates. If these strings should be plotted as numbers, cast to the appropriate data type before plotting.
```

**Comment by bobleesj** [comment]
<img width="990" alt="Screenshot 2024-07-06 at 11 35 54 AM" src="https://github.com/bobleesj/cifkit/assets/14892262/e7a48303-617e-4200-9897-e6a0fb1a9ea7">

The coordination number x-axis should integer values.

**Comment by bobleesj** [comment]
> For the histogram generations with coordination numbers, the following warning should be removed.

This is fixed in 1.0.2

> Furthermore, how is it possible to have such a low value CN value of around 3? I need to investigate this.

This is not yet discussed.

**Comment by bobleesj** [comment]
<img width="459" alt="Screenshot 2024-07-09 at 7 30 15 PM" src="https://github.com/bobleesj/cifkit/assets/14892262/98f728f5-54cc-46e8-91f3-9134761a8ebc">

<img width="455" alt="Screenshot 2024-07-09 at 7 30 21 PM" src="https://github.com/bobleesj/cifkit/assets/14892262/a340f36a-35e5-432e-8d77-9dcf91df02f1">

```python
# y-axis
plt.gca().yaxis.set_major_locator(MaxNLocator(integer=True))

# x-axis
int_keys = sorted(map(int, keys))
plt.xticks(
range(min(int_keys), max(int_keys) + 1)
)
plt.gca().xaxis.set_major_locator(MaxNLocator(integer=True))
```

x- and y-axis integer problem is fixed.
